### DISCUSSION TITLE
Renovate created an onboarding PR for a repo I didn't configure to use with it

### DISCUSSION THREAD

**Discussion by olegkrivtsov** [renovatebot/renovate#12508] [Request Help] [answered]
### How are you running Renovate?

WhiteSource Renovate hosted app on github.com

### Please select which platform you are using if self-hosting.

github.com

### Describe the bug

Today I received a notification from GitHub that Renovate created an onboarding PR for my repo https://github.com/olegkrivtsov/using-zend-framework-3-book although I have no `renovate.json` file in that repo.

The PR link: https://github.com/olegkrivtsov/using-zend-framework-3-book/pull/91

Could that be a bug?

### Have you created a minimal reproduction repository?

No reproduction, but I have linked to a public repo where it occurs

**Comment by rarkins** [comment]
The point of onboarding is to create a renovate.json

**Comment by olegkrivtsov** [comment]
Hi @rarkins sorry, I probably explained in a wrong way. 

I agree that the onboarding would create renovate.json. But Renovate shouldn't have created an onboarding PR for that particular repo since I didn't command it to do so. I've just checked my GitHub Renovate App settings and for some reason that particular repo was in the list of repos Renovate has access to. 

So this is not a bug, but probably my misunderstanding. Although it is a bit strange why Renovate waited for 3 months before it created the onboarding PR in that repo.

**Comment by viceice** [comment]
@olegkrivtsov The [`onboardingNoDeps`](https://docs.renovatebot.com/self-hosted-configuration/#onboardingnodeps) settings was enabled on hosted app recently.

**Comment by rarkins** [comment, marked as answer]
It's a result of the `onboardingNoDeps` feature you implemented. It is now enabled in the hosted app any time user "selects" individual repos for the app instead of chooses "All repositories". If you have chosen "Select repositories" when installing/configuring the app, and you have selected a particular repo - whether by accident or not - it's a very clear signal that you want Renovate running on it.
